Makefile

```diff
@@ -397,17 +398,17 @@ crd-ref-docs: $(CRD_REF_DOCS) #EXHELP Generate the API Reference Documents.
 VENVDIR := $(abspath docs/.venv)
 
 .PHONY: build-docs
-build-docs: venv
+build-docs:
 	. $(VENV)/activate; \
 	mkdocs build
 
 .PHONY: serve-docs
-serve-docs: venv
+serve-docs:
 	. $(VENV)/activate; \
 	mkdocs serve
 
 .PHONY: deploy-docs
-deploy-docs: venv
+deploy-docs:
```

Comment:
What's the story here? `Makefile.venv` has some doc text that makes this sounds important.

https://github.com/perdasilva/operator-controller/blob/3747b8637c3c24671917c1e14b6c4ca406df35bc/Makefile.venv#L15-L17

Reply:
my bad - ide was complaining so I thought they were left over from a few months back when there was some makefile refactoring. I've put them back in.